models/import.js

```diff
@@ -90,6 +95,24 @@ class TrelloCreator {
       stars: 0,
       title: trelloBoard.name,
     };
+    // now add other members
+    if(trelloBoard.memberships) {
+      trelloBoard.memberships.forEach((trelloMembership) => {
+        const trelloId = trelloMembership.idMember;
+        // do we have a mapping?
+        if(this.members[trelloId]) {
+          const wekanId = this.members[trelloId];
+          // do we already have it in our list?
+          if(!boardToCreate.members.find((wekanMember) => {return (wekanMember.userId === wekanId);})) {
```

Comment:
Style note: I would write it either:

```es6
if (!boardToCreate.members.find((wekanMember) => {
  return (wekanMember.userId === wekanId);
})) {
```

or

```es6
if (!boardToCreate.members.find((wekanMember) => wekanMember.userId === wekanId)) {
```

With a preference for the second one.